xain_fl/coordinator/coordinator.py

```diff
@@ -393,8 +388,22 @@ def _handle_end_training_round(
         # submitting the updates and raise an exception if it is the wrong
         # round.
 
-        # record the request data
-        model_weights: ndarray = proto_to_ndarray(message.weights)
+        # HACK: Participants are currently not aware of they ID the
+        # coordinator uses to identify them, so they generate a UUID
+        # and use it to upload their results. The EndTrainingMessage
+        # contains that UUID, so that the coordinator can retrieve the
+        # weights.
+        #
+        # FIXME: handle the case where the participant didn't send a
```

Comment:
Would be great to have a ticket mentioned here, so we would not forget about it.

Reply:
done: https://xainag.atlassian.net/browse/PB-436

Reply:
thanks